packages/react-cosmos-webpack/src/__tests__/webpack-config.js

```diff
@@ -225,3 +225,81 @@ describe('loaders', () => {
     });
   });
 });
+
+describe('output', () => {
+  beforeAll(() => {
+    mockCosmosConfig = {
+      componentPaths: ['src/components'],
+      fixturePaths: ['test/fixtures'],
+      ignore: [],
+      globalImports: ['./global.css'],
+      hot: true,
+      outputPath: '__mock__outputPath'
+    };
+  });
+  describe('with shouldExport false', () => {
```

Comment:
It's annoying that I'm enforcing this without having an ESLint rule, but could you please add a newline between each `describe` and `test` blocks? The extra air improves readability.